llama2.mojo

```diff
@@ -937,12 +939,12 @@ fn print_str(s: PointerString):
     if (s[1].to_int() == ord("0")) and (s[2].to_int() == ord("x")):
         var d1: Int = s[3].to_int()
         var d2: Int = s[4].to_int()
-        print_no_newline(chr(str2num(d1) * 16 + str2num(d2)))
+        print(chr(str2num(d1) * 16 + str2num(d2)))
```

Comment:
Since `print_no_newline` was removed the correct way to mimic this now is with a second argument `end=""` otherwise the default behaviour adds the newline character.